databricks_cli/configure/config.py

```diff
@@ -58,6 +58,14 @@ def get_profile_from_context():
     return context_object.get_profile()
 
 
+def debug_option(f):
+    def callback(ctx, param, value): #  NOQA
+        context_object = ctx.ensure_object(ContextObject)
+        context_object.set_debug(value)
+    return click.option('--debug', '-d', is_flag=True, callback=callback,
```

Comment:
Probably we do not need "-d" here? It is okey to require the full option name for debug